packages/react-storage/src/components/StorageBrowser/componentsDefault.tsx

```diff
@@ -123,7 +129,13 @@ const Navigation: Components['Navigation'] = ({ items }) => {
 
 const LoadingIndicator: Components['LoadingIndicator'] = ({ isLoading }) => {
   if (isLoading) {
-    return <Loader variation="linear" size="small" />;
+    return (
+      <Loader
+        className="amplify-storage-browser__loader"
```

Comment:
should this be `STORAGE_BROWSER_BLOCK`?

Reply:
You are right, but I don't want to wait for another round of builds